models/category/src/category-draft/presets/sample-data-goodstore/bedroom-furniture.ts

```diff
@@ -0,0 +1,31 @@
+import {
+  LocalizedString,
+  KeyReference,
+} from '@commercetools-test-data/commons';
+import type { TCategoryDraftBuilder, TCategoryDraft } from '../../../types';
+import empty from '../empty';
+import furniture from './furniture';
+
+const furnitureDraft = furniture().build<TCategoryDraft>();
+
+const bedroomFurniture = (): TCategoryDraftBuilder =>
+  empty()
+    .name(
+      LocalizedString.presets
+        .empty()
+        ['en-GB']('Bedroom Furniture')
+        ['en-US']('Bedroom Furniture')
+        ['de-DE']('Schlafzimmer')
+    )
+    .slug(
+      LocalizedString.presets
+        .empty()
+        ['en-GB']('bedroom-furniture')
+        ['en-US']('bedroom-furniture')
+        ['de-DE']('schlafzimmer')
+    )
+    .key('bedroom-furniture')
+    .orderHint('9')
```

Comment:
This appears to be a typo in the original data, legal values are from 0 to 1

Reply:
fixed with [439a32c](https://github.com/commercetools/test-data/pull/357/commits/439a32c559fb3c5e19eec43ab4b2a07dc184ff10)